### PR TITLE
Update release note with conditional breakpoints support

### PR DESCRIPTION
**Pull request by praveennadarajah**
## Purpose
This PR will update the release note with conditional breakpoint support.

### CODE REVIEW

File: release-notes/swan-lake-alpha3-release-note.md [hunk 1]
```diff
@@ -123,4 +123,6 @@ io:println(nameText); // "DanBrown"
 
 ##### Debugger
 
+Conditional breakpoints are now supported by ballerina debugger.
```
[the remark this answers is not in the document]
Comment (praveennadarajah) [reply release-notes/swan-lake-alpha3-release-note.md]: Fixed

[the remark this answers is not in the document]
Comment (NipunaRanasinghe) [reply release-notes/swan-lake-alpha3-release-note.md]: @MaryamZi +1 for the suggestion

@praveennadarajah should we also be bit more descriptive about the feature if possible? maybe we can add one more line saying "with this feature, users can now configure conditions for ballerina breakpoints in vscode debug view". @MaryamZi @praveennadarajah WDYT?

[the remark this answers is not in the document]
Comment (praveennadarajah) [reply release-notes/swan-lake-alpha3-release-note.md]: @MaryamZi @NipunaRanasinghe @praneesha So shall we finalize to 
`The debugger now supports conditional breakpoints and with this feature, users can now configure conditional expressions for ballerina breakpoints in vscode debug view` ?

**Review by praneesha — APPROVED** [review]

File: release-notes/swan-lake-alpha3-release-note.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MaryamZi) [reply release-notes/swan-lake-alpha3-release-note.md]: How about

"The debugger now supports conditional breakpoints. Users can now configure conditional expressions for Ballerina breakpoints in the VSCode debug view"

[the remark this answers is not in the document]
Comment (NipunaRanasinghe) [reply release-notes/swan-lake-alpha3-release-note.md]: +1 :slightly_smiling_face:

[the remark this answers is not in the document]
Comment (praneesha) [reply release-notes/swan-lake-alpha3-release-note.md]: "Now, the debugger supports conditional breakpoints. You can configure conditional expressions for Ballerina breakpoints in the VSCode debug view."

[the remark this answers is not in the document]
Comment (praveennadarajah) [reply release-notes/swan-lake-alpha3-release-note.md]: @NipunaRanasinghe @MaryamZi Shall we finalize the above?
`Now, the debugger supports conditional breakpoints. You can configure conditional expressions for Ballerina breakpoints in the VSCode debug view.`

[the remark this answers is not in the document]
Comment (praneesha) [reply release-notes/swan-lake-alpha3-release-note.md]: +1

[the remark this answers is not in the document]
Comment (praveennadarajah) [reply release-notes/swan-lake-alpha3-release-note.md]: Done

**Pull request merged**